install/migrations/update_10.0.7_to_10.0.8/entity.php

```diff
@@ -41,17 +41,17 @@
     Entity::getTable(),
     'latitude',
     'latitude',
-    "decimal(9,6) NOT NULL DEFAULT '0.0000'",
+    "decimal(9,6) DEFAULT NULL",
```

Comment:
> Also I don't remember how many decimals open street map handles and if the definition you use (9,6) is correct for backward compatibility

After some digging : 
* [openstreetmap node doc](https://wiki.openstreetmap.org/wiki/Node) says it handles 7 digits for the fractional part. 
* after tests : the "define location here" button on a map can return much more digits. By zooming on the map i can define a location with lat/long with 14 digits for the fractional part.
* users may have recorded location by reporting data from googlemap for example which can also use more than 6 digits for the fractional part

I don't know the rule of precision we want to apply but today, there is none in GLPI such as user can store very "accurate" location.

By changing the type for `Decimal(9,6)`, we will round some data in a such way we will change a behavior and maybe break some user data.
In this PR, we should change the manual input according to the selected rule of precision to disallow contribution of decimal with to much precision. We should also change how it is contributed with the "change location to here" button.

It should not be a "fix" but an enhancement for next 10.1 version.

Reply:
Yes, required changes should target main branch for 10.1 release.